modules/installation-configuration-parameters.adoc

```diff
@@ -427,14 +430,19 @@ The value can also be the name of an existing Glance image, for example `my-rhco
 }
 ----
 
+|`platform.openstack.externalDNS`
+|IP addresses for external DNS servers that cluster instances use for DNS resolution.
+|A list of IP addresses as strings. For example `["8.8.8.8", "192.168.1.12"]`.
```

Comment:
Comma after `For example`, as was done in the previous config param additions.

Reply:
TY.